src/Filters/HttpFilter.php

```diff
@@ -132,15 +134,25 @@ protected function addFiltersToQuery(Builder $builder)
     protected function filtersExact(Builder $query, $value, string $property): Builder
     {
         $property = self::sanitize($property);
-
+    
+        if ($query->getModel()->hasCast($property, ['date', 'datetime', 'immutable_date', 'immutable_datetime']) || $property === $query->getModel()->getCreatedAtColumn() || $property === $query->getModel()->getUpdatedAtColumn()) {
```

Comment:
what about deleted_at? maybe these columns already has casts on them and we don't need to check the name explicitly?

Reply:
no need, its a datetime cast by default:
```
# SoftDeletes.php
public function initializeSoftDeletes()
    {
        if (! isset($this->casts[$this->getDeletedAtColumn()])) {
            $this->casts[$this->getDeletedAtColumn()] = 'datetime';
        }
    }
```